app/code/AstoundDRudenko/PriceBadge/view/frontend/web/js/product/mixin/price-badges.js

```diff
@@ -0,0 +1,105 @@
+define([
+    'jquery',
+    'underscore',
+    'mage/template',
+    'mage/smart-keyboard-handler',
+    'mage/translate',
+    'priceUtils',
+    'jquery/ui',
+    'jquery/jquery.parsequery',
+    'mage/validation/validation'
+], function ($, _, mageTemplate, keyboardHandler, $t, priceUtils) {
+    'use strict';
+
+    return function (widget) {
+        $.widget('mage.SwatchRenderer', widget, {
+            /**
+             * Add possibility to hide/show price badges
+             *
+             * @private
+             */
+            _UpdatePrice: function () {
+                var $widget = this,
```

Comment:
you declare $widget as this, but than you are using both $widget variable and 'this'. Is there any purpose for that?